### DISCUSSION TITLE
Logout feature

### DISCUSSION THREAD

**Discussion by l3all** [linuxserver/Heimdall#1096] [Ideas]
It would be necessary to have a logout feature when using the dashboard outside the network.

**Comment by keriati** [comment]
I think the Switch user button should log you out right now.

**Reply by bcutter** [reply]
...which you only have if there's more than one user. So not really a solution comparable to a rock-solid simple log out button.

**Comment by adibindrianto** [comment]
It seems like the feature isn't available yet, are there really no plans to add?
